opentelemetry-api/setup.py

```diff
@@ -40,6 +40,15 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
+    entry_points={
+        "console_scripts": [
```

Comment:
I'm strongly in favor of moving these to a new distribution under `ext`.

Reply:
Well, yes, unless auto instrumentation is something that ends up being defined in the specification, it should not be in the API package. Nevertheless, `ext` does not seem to be the place to put this in either, so far it contains "extensions" (this word is too broad, I think) that make it possible to easily instrument different software components.

The autoinstrumentation can be a feature of our SDK, maybe, or it can be a separate Python package which could itself have an `ext` directory where we could put what we already have in `ext`.

Reply:
OK, I should have said "I'm strongly in favor of moving these to a new distribution". I don't really mind a new top-level package either, it doesn't have to be under `ext/`.

Reply:
+1 on moving the console script to a separate package. Maybe it ends up moving into the https://github.com/open-telemetry/opentelemetry-auto-instr-python/ repo, just a thought